### DISCUSSION TITLE
Deeplinking does not work. I always just land on the home page.

### DISCUSSION THREAD

**Discussion by philipheinser** [community/community#44965] [Mobile]
<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.96.0 (88967961)
Device: iPhone 12 mini (iOS 16.2)
</details>

**Comment by jge162** [comment]
@philipheinser for deeplinking requires proper setup and configuration, so if it is not implemented correctly, it will not work. it only works if the app is installed on the user's device. If the app is not installed, the user will be directed to the app store to download it. But, if the app is installed, it must be configured to handle deeplinks. If this is not done properly, the app will not be able to handle the deeplink and the user will be directed to the home page. In addition, may only work on certain types of devices or operating systems, so if the user's device is not supported, the deeplink will not work.

check you are using correct URL and the URL should be properly encoded.

by some odd chance too, Deeplink can be blocked by the OS or browser security feature.

**Reply by philipheinser** [reply]
https://user-images.githubusercontent.com/3705499/213861432-a3aa933c-e09f-4577-adf1-ce9e70011cd0.MOV

**Reply by philipheinser** [reply]
The app is installed and also the os detects the link and redirects to the app. But the GitHub app does not handle it correctly and does not open the repo.

**Reply by jge162** [reply]
@philipheinser You can also try testing the deeplink on different devices and operating systems to see if the issue is specific to one device or OS.

**Reply by jge162** [reply]
btw I never use app. I deleted.

**Reply by jge162** [reply]
Was I able to answer your question or you still stuck? @philipheinser if not lmk where u are stuck. Ty

**Reply by philipheinser** [reply]
I did not had any question I just wanted to report a bug.

**Comment by jge162** [comment]
Gotchu sounds good. Great post!

**Comment by bazo** [comment]
yep, this is pretty annoying. but sometimes clicking repo links in twitter or on web opens correctly in the mobile app
